controllers/config/defaults.go

```diff
@@ -56,8 +49,41 @@ const (
 	ObjectStorageSecretKey      = "secretkey"
 )
 
-// Default ResourceRequirements
+// DSPO Config File Paths
+const (
+	APIServerImagePath            = "images.apiServer"
+	APIServerArtifactImagePath    = "images.artifact"
+	PersistenceAgentImagePath     = "images.persistentAgent"
+	ScheduledWorkflowImagePath    = "images.scheduledWorkflow"
+	ViewerCRDImagePath            = "images.viewerCRD"
+	APIServerCacheImagePath       = "images.cache"
+	APIServerMoveResultsImagePath = "images.moveResultsImage"
+	MlPipelineUIImagePath         = "images.mlPipelineUI"
+	MariaDBImagePath              = "images.mariaDB"
+	MinioImagePath                = "images.minio"
```

Comment:
I think the convention is to use PascalCase for viper configs, right?

Reply:
I'm not sure why Viper would impose a convention on how applications handle their config files, afaik the convention for yaml files is to do do camelcase and that's what I tried to adhere to.

Reply:
I updated it to pascal case regardless, this is what dspipelines app does it, might as well stay consistent with that.